pipettes/core/peripheral_tasks.cpp

```diff
@@ -11,17 +11,17 @@ static auto i2c3_task_client =
     i2c::writer::Writer<freertos_message_queue::FreeRTOSMessageQueue>();
 
 static auto i2c1_task_builder =
-    freertos_task::TaskStarter<512, i2c::tasks::I2CTask>{};
+    freertos_task::TaskStarter<256, i2c::tasks::I2CTask>{};
 static auto i2c3_task_builder =
-    freertos_task::TaskStarter<512, i2c::tasks::I2CTask>{};
+    freertos_task::TaskStarter<256, i2c::tasks::I2CTask>{};
 template <template <typename> typename QueueImpl>
 using PollerWithTimer =
     i2c::tasks::I2CPollerTask<QueueImpl, freertos_timer::FreeRTOSTimer>;
 
 static auto i2c1_poll_task_builder =
-    freertos_task::TaskStarter<1024, PollerWithTimer>{};
+    freertos_task::TaskStarter<512, PollerWithTimer>{};
```

Comment:
hmm I think this was increased for a specific reason, but I can't remember why. We'll just need to do a lot of testing with the sensors

Reply:
I'm happy to preemptively bump this one back up, I just indiscriminately cut everything in half.